### DISCUSSION TITLE
日记页面下标签Emoji显示问题

### DISCUSSION THREAD

**Discussion by ZhangWeilong** [Tokinx/Wing#123] [问答]
![image](https://github.com/Tokinx/Wing/assets/8246653/a06b0569-396f-4c2f-b668-14b886450d20)

在日记页面下，添加含有Emoji的标签，可以正常显示，但右侧不会显示这个标签。右侧标签中带有Emoji是如何实现的。

**Comment by 37duer** [comment]
是的，我也是遇到这个问题，但自带的那些图标就可以

**Comment by 37duer** [comment]
我找到原因了，那是因为你第一次建的时候没有加表情，所以在数据库里没写入，自己去数据库改吧
[表: wp_terms]

**Reply by ZhangWeilong** [reply]
这个问题我解决了，我是因为数据库编码问题，修改为utf8mb4就正常了
